editor/components/toolbar/index.js

```diff
@@ -9,29 +9,32 @@ import classNames from 'classnames';
 import './style.scss';
 import IconButton from 'components/icon-button';
 
-function Toolbar( { controls } ) {
```

Comment:
FYI there are no changes to this component's render logic. It needed to be converted from a stateless functional component to a class component to support `ref` being applied by `Editable`.

See: https://facebook.github.io/react/docs/refs-and-the-dom.html#refs-and-functional-components